src/resources/views/cart/default.blade.php

```diff
@@ -31,6 +31,8 @@
                 dusk="v-cartitem-component"
             ></mx-cart-item>
 
+            @includeIf('cart.upsells')
```

Comment:
Always a bit unsure if things like this should be extended in the vendor views, e.g. `maxfactor::cart.upsells`. I feel like it keeps things more contained? What are you thoughts @tristanward ?

Reply:
Yea good shout, not sure why but that does feel better.  I'll update!